source/includes/_tipser-elements.md

```diff
@@ -299,6 +300,10 @@ The `onError` event handler is used with `useDefaultErrorHandler` config option.
 
 <aside class="warning">This section requires simplification. It isn't clear how to use event handler with combination of this configuration</aside>
 
+#### onStockCountChange
+
+This handler takes care of the edge case, when while in the checkout process and before payment, an item in the cart, which has a low stock count, becomes unavailable (stockCount = 0). By default, in such a situation, we display an overlay with an information about the stock count change and the button for reloading the checkout. If. however, you wish to customize this behaviour in your code, you can use a callback `onStockCountChange(items: CartItemModel[]) => void`, with `CartItemModel` as type. If usage of this event handler is detected by Tipser Elements in your code, the default behaviour will not occur. 
```

Comment:
Mała nieścisłość odnośnie kiedy ten event jest triggerowany (w rzeczywistości leci kiedy  0 < stockCount < quantity)

Reply:
A few suggestions how to express it more succintly:

If~~. however,~~ you wish to customize this behaviour ~~in your code~~, you can use a callback `onStockCountChange(items: CartItemModel[]) => void`~~, with `CartItemModel` as type. If usage of this event handler is detected by Tipser Elements in your code, the default behaviour will not occur. ~~, which will prevent the default behavior.